File: comparacao_python/get_set/Informa_Novo_Automovel.py
from Novo_Automovel import novo_automovel

class Informa_Novo_Automovel:
    @staticmethod
    def main():
        novo_carro = novo_automovel()

        novo_carro.set_marca= input("Qual o marca do carro?\t")
        novo_carro.set_modelo= input("Qual o modelo do carro?\t")
        novo_carro.set_cor= input("Qual a cor do carro?\t")
        novo_carro.set_preco= float(input("Qual o preço do carro?\t"))

        print("\nDados do automóvel:\n")
        print(f"Marca:{novo_carro.marca}\t")
        print(f"Modelo:{novo_carro.modelo}\t")
        print(f"Cor:{novo_carro.cor}\t")
        print(f"Preco:{novo_carro.preco}\t")

if __name__ == "__main__":
    Informa_Novo_Automovel.main()

File: exec/src/get_set/Informa_Novo_Automovel.java
package get_set;
import javax.swing.JOptionPane;

public class Informa_Novo_Automovel {
    public static void main(String []args) {
        Novo_Automovel a = new Novo_Automovel();

        a.setMarca(JOptionPane.showInputDialog("Qual a marca do carro?"));
        a.setModelo(JOptionPane.showInputDialog("Qual o modelo do carro?"));
        a.setCor(JOptionPane.showInputDialog("Qual a cor do carro?"));
        a.setPreco(Double.parseDouble(JOptionPane.showInputDialog("Qual o preco do carro?")));

        System.out.println("marca do carro:\t"+ a.getMarca());
        System.out.println("modelo do carro:\t"+ a.getModelo());
        System.out.println("cor do carro:\t"+ a.getCor());
        System.out.println("preco do carro:\t"+ a.getPreco());
    }
}
